src/Amazon.Extensions.CognitoAuthentication/CognitoUser.cs

```diff
@@ -554,6 +554,62 @@ public virtual async Task<ListDevicesResponse> ListDevicesV2Async(int limit, str
             return await Provider.ListDevicesAsync(listDevicesRequest, cancellationToken).ConfigureAwait(false);
         }
 
+        /// <summary>
+        /// Request code for authenticator app.
+        /// </summary>
+        /// <returns><see cref="AssociateSoftwareTokenResponse"/> with secret code.</returns>
+        public virtual async Task<AssociateSoftwareTokenResponse> AssociateSoftwareTokenAsync()

```

Comment:
Can you add versions with `cancellationToken` for these methods?

Reply:
Done.